### PR TITLE
Add selectAllEvent keyboard listener event (default is ctrl+A)

### PR DESCRIPTION
**Pull request by Eng-Elias**
in this branch I enabled SelectAll keyboard event and I did some changes on selectAllHotkey method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -283,6 +283,8 @@ A callback executed after a search event. This is called once on initial render
 
 #### `copyEvent (=(e: KeyboardEvent) => e.key === "c" && (e.metaKey || e.ctrlKey))`
 
+#### `selectAllEvent (=(e: KeyboardEvent) => e.key === "a" && (e.metaKey || e.ctrlKey))`
```
Comment (jjti) [inline README.md:288]: I think we'll want to move this to just below the existing description about `ctrl+c` and copy, and describe it's default keybinding `ctrl+a`

**Comment by jjti** [comment]
Thanks @Eng-Elias and this looks great. I think there's some small linting changes we'll want to address: https://github.com/Lattice-Automation/seqviz/actions/runs/5819124159/job/15844201872?pr=223 (`npm run fix` should help)

File: README.md (hunk 1 above)
Comment (Eng-Elias) [reply README.md:288]: fixed in 596d3ea51445930629df90b271a07c46ddcf1db7

**Comment by Eng-Elias** [comment]
Thanks @jjti, 
I fixed the linting in 596d3ea51445930629df90b271a07c46ddcf1db7 , I hope this will make the pull request ready. 

I also have some enhancements on SeqViz, I will work on them on another branch and I will open another pull request so you can take a look on them and give me your opinion.

**Review by jjti — APPROVED** [review]

**Pull request merged**